presto-hive/src/test/java/com/facebook/presto/hive/TestPrestoS3FileSystem.java

```diff
@@ -169,6 +176,62 @@ public void testReadForbidden()
     }
 
     @Test
+    public void testCreateWithNonexistentStagingDirectory()
+        throws Exception
+    {
+        java.nio.file.Path tmpdir = Paths.get(StandardSystemProperty.JAVA_IO_TMPDIR.value());
+        java.nio.file.Path stagingParent = Files.createTempDirectory(tmpdir, "staging");
```

Comment:
Name the prefix `"test"` just to be clear that it is different from the `"staging"` directory below